eng/native/build-commons.sh

```diff
@@ -392,6 +392,10 @@ while :; do
             __TargetArch=wasm
             ;;
 
+	ppc64le|-ppc64le)
```

Comment:
spaces for indentation:
```suggestion
        ppc64le|-ppc64le)
```